gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,135 @@
 ---
-description: Proof Market front end guide
+description: Web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
 
-{% hint style="info" %}
-Please note this product is in beta and expected to change.
-{% endhint %}
-
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
-
-## New user signup
-
-All users who wish to interact with the proof market must create an account.
+The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
 
 {% hint style="info" %}
-If you do not have the credentials, users can still view the exchange, but they will be unable to post orders to buy/sell proofs.
+Please note this product is in beta and expected to change.
 {% endhint %}
 
-<figure><img src="../.gitbook/assets/image (6).png" alt=""><figcaption></figcaption></figure>
+## Creating an account 
 
-Click on Sign up
+To interact with the Proof Market, create an account at [proof.market](https://proof.market)
+or [through the command-line interface](cmd-reference/user.md).
+Without an account, you can view the exchange, but you won't be able to post orders 
+to buy or sell proofs.
 
-<figure><img src="../.gitbook/assets/image.png" alt=""><figcaption></figcaption></figure>
+To register a new user via the web interface, click **Sign up**:
```

Comment:
Please check this out, I've updated this line according to the current design of proof.market